PrivateCollections/Ethiopia/MarigetaDamdaEndalaw/MD004.xml

```diff
@@ -0,0 +1,147 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="MD004" xml:lang="en" type="mss">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Lǝssāna sabʾ</title>
```

Comment:
Very nice record, Addissie! May be you can give the title both in Geez and English? This is a new fashion Denis suggested :-)

Reply:
I do not know about the fashion Denis suggested but we normally only provide full title in Fidal and in translation for works, not for manuscripts.

Reply:
> I do not know about the fashion Denis suggested but we normally only provide full title in Fidal and in translation for works, not for manuscripts.

Ok! then we follow this! @Addisie, you can ignore my suggestion as clearified by Eugenia.